### PR TITLE
THEEDGE-2149 Logging format fixes

### PR DESCRIPTION
**Pull request by tpapaioa**
# Description
This PR fixes some typos in logging messages. It also changes logging calls that log the specific variable values as part of the message, so that the variable data are in extra fields instead. This is to bring them in line with the contributor guidelines:

```
- Messages will only contain plain English and we should be able to group by messages, which means that no variables can be added.
- All useful data must go onto extra fields. Try to keep the pattern for the extra field name from the rest of the code for common things ("error" for errros, "imageID" for image ID, or for other objects use "objectID").
```

Fixes # (THEEDGE-2149)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes
- [x] I run `make lint` to prints out coding style mistakes and fix them

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
07e0cd0c-ed59-4574-84fe-1357998a5b70
2957d7a7-e159-4a2c-b1bc-f892a7e28a1b

**Review by loadtheaccumulator — APPROVED** [review]
/lgtm
